app/routes/__init__.py

```diff
@@ -0,0 +1,150 @@
+from typing import Any, Dict, Tuple, Union
+
+import mercantile
+import pendulum
+from fastapi import Depends, HTTPException, Path
+from fastapi.logger import logger
+from shapely.geometry import box
+
+from ..crud.sync_db.vector_tile_assets import get_dynamic_versions, get_static_versions
+from ..models.enumerators.dynamic_enumerators import (
+    get_dynamic_datasets,
+    get_static_datasets,
+)
+
+DATE_REGEX = "^\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$"  # mypy: ignore
+VERSION_REGEX = r"^v\d{1,8}\.?\d{1,3}\.?\d{1,3}$|^latest$"
+NOW = pendulum.now()
+DEFAULT_START = NOW.subtract(weeks=1).to_date_string()
+DEFAULT_END = NOW.to_date_string()
+
+Geometry = Dict[str, Any]
+Bounds = Tuple[float, float, float, float]
```

Comment:
You define these types in a couple places, is it possible to just define them from one place?